controllers/admin/AdminPaypalOnboardingPrestashopCheckoutController.php

```diff
@@ -23,8 +23,14 @@
 
 class AdminPaypalOnboardingPrestashopCheckoutController extends ModuleAdminController
 {
+    /**
+     * @var Ps_checkout
+     */
+    public $module;
+
     public function init()
     {
+        parent::init();
```

Comment:
Why we don't call init parent function before ? Because without this, we can't use the service container from the core

Reply:
https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/Controller.php#L169 https://github.com/PrestaShop/PrestaShop/blob/develop/classes/controller/AdminController.php#L2769 are the parents' calls